app/js/auth/index.js

```diff
@@ -7,7 +7,8 @@ import cookie from 'js-cookie';
 
 class Auth {
   andelaLoginUrl() {
-    return `${cookie.get(CVar.auth_url)}?redirect_url=http://${
+    const auth_url = cookie.get(CVar.auth_url) || 'http://api-staging.andela.com/login';
```

Comment:
We had the `auth_url` in the environment variable, which was supposed to be set in the cookie, so we try to fetch it from the cookie again, this doesn't work